database/database.go

```diff
@@ -77,6 +78,35 @@ func CreateMySqlConnection(ctx context.Context, dbConfig DatabaseConfig) (*sqlx.
 	return db, nil
 }
 
+// CreateOracleConnection creates a connection to a oracle database
+func CreateOracleConnection(ctx context.Context, dbConfig DatabaseConfig) (*sqlx.DB, error) {
+
+	log.Println("Connecting to database...")
+
+	conn := fmt.Sprintf(
```

Comment:
Use the built in go_ora method BuildUrl to create the connection string, for example"

conn := go_ora.BuildUrl(Host, Port, Name, User, Password, nil)